env-bootstrap/src/lib.rs

```diff
@@ -11,6 +11,33 @@ pub fn set_wezterm_executable() {
     }
 }
 
+pub fn fixup_snap() {
+    if std::env::vars_os().any(|(key, _)| key == "SNAP") {
```

Comment:
This feels a bit clearer and easier to grok:

```suggestion
    if std::env::var_os("SNAP").is_some() {
```

Reply:
Oooh, yeah. That's better. I didn't notice that API! :grin: